src/runtime/pkg/katatestutils/utils.go

```diff
@@ -320,6 +321,7 @@ func MakeRuntimeConfigFileData(config RuntimeConfigOptions) string {
 	msize_9p = ` + strconv.FormatUint(uint64(config.DefaultMsize9p), 10) + `
 	enable_debug = ` + strconv.FormatBool(config.HypervisorDebug) + `
 	guest_hook_path = "` + config.DefaultGuestHookPath + `"
+    guest_hook_timeout = ` + strconv.FormatUint(uint64(config.DefaultGuestHookTimeout), 10) + `
```

Comment:
please go fmt the code.

Reply:
This is already the result of a `go fmt`, but it's strange why GitHub shows it like this.